docs/create.md

```diff
@@ -8,22 +8,22 @@ To create and publish an event to the activity app, a new `IEvent` should be fet
 // When you have a plain file without a class, you can use
 // \OC::$server->getActivityManager() instead.
 
-$event = $this->activityManager->createEvent();
+$event = $this->activityManager->generateEvent();
 ...
-$this->activityManager->->publish($event);
+$this->activityManager->publish($event);
 ```
 
 The following values **must** be set before publishing an event:
 
 * `setApp()`
-* `setType()` - this must match an `ISetting::getIdentifier()`
+* `setType()` - this must match an `\OCP\Activity\ISetting::getIdentifier()`
 * `setAffectedUser()`
-* `setAuthor()`
-* `setTimestamp()`
 * `setSubject()`
-* `setObject()`
 
 Additionally these values **can** be set:
+* `setAuthor()` - if no author is set, the current user will be used
+* `setTimestamp()` - if no not set, the current time will be used
+* `setObject()`
```

Comment:
This is just not enforced to break old activities, but it should be in the list above so we can at somepoint say "okay ignore all items without objects, the future is here"